app/src/main/java/com/framgia/oleo/screen/login/LoginFragment.kt

```diff
@@ -132,7 +147,26 @@ class LoginFragment : BaseFragment(), View.OnClickListener {
                 context!!, editTextPassword.text.toString(), textLayoutPassWord
             )
         ) {
-            replaceFragment(R.id.containerMain, HomeFragment.newInstance())
+            signWithPhoneNumberAndPassword(
+                editTextPhoneNumber.text.toString(),
+                object : ValueEventListener {
+                    override fun onDataChange(p0: DataSnapshot) {
```

Comment:
```suggestion
                    override fun onDataChange(dataSnapShot: DataSnapshot) {
```
đặt lại mấy cái tên này cho dễ hiểu bạn ơi